docs/data/pages.ts

```diff
@@ -47,10 +47,15 @@ const pages = [
           },
           { pathname: '/x/api/data-grid/grid-row-params', title: 'GridRowParams' },
           { pathname: '/x/api/data-grid/grid-row-spacing-params', title: 'GridRowSpacingParams' },
+          { pathname: '/x/api/data-grid/grid-csv-export-options', title: 'GridCsvExportOptions' },
           {
             pathname: '/x/api/data-grid/grid-print-export-options',
             title: 'GridPrintExportOptions',
           },
+          {
+            pathname: '/x/api/data-grid/grid-excel-export-options',
```

Comment:
Maybe link to this page in Excel export docs?
Also, it seems to be missing in left nav sidebar